openedx/core/djangoapps/content/course_overviews/tests/test_signals.py

```diff
@@ -29,13 +32,32 @@ class CourseOverviewSignalsTestCase(ModuleStoreTestCase):
     TODAY = datetime.datetime.utcnow().replace(tzinfo=UTC)
     NEXT_WEEK = TODAY + datetime.timedelta(days=7)
 
+    def assert_changed_signal_sent(self, changes, mock_signal):
```

Comment:
This is unchanged, just moved. Having a verification function in the middle of other unit tests breaks my brain sometimes (organizationally), so I moved it above the other unit tests.